### PR TITLE
created navigation bar with corresponding pages

### PR DESCRIPTION
**Pull request by Anusha-Acharya-99**
Created homepage with navbar which contains site name and items(home, series, package, signin, sign out) by clicking on which opens corresponding pages. Also created sub navbar in series page with items(current, future).

### CODE REVIEW

File: components/navbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          <Link href="/signin">Sign in</Link>
+        </li>
+        <li>
+          <Link href="/signout">Sign out</Link>
+        </li>
+      </ul>
+    </div>
+  );
+};
+
```
Comment (nikhilrobosoft) [inline components/navbar/index.tsx:30]: Link tag is not required. Next js works differently. The pages folder is responsible for routing. This should only contain the <li> tags, now futher this component should be used in the pages/{respective route files} to render the data.

File: components/subnavbar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      <li>
+        <Link href="/series/current">Current</Link>
+      </li>
+      <li>
+        <Link href="/series/future">Future</Link>
+      </li>
+    </ul>
+  );
+};
+
```
Comment (nikhilrobosoft) [inline components/subnavbar/index.tsx:16]: Similar to the above navbar component here <link> is not required, only <li> should be there. This subnav component can be used in the current & future route files under pages. 

return (
        <Navbar>
         <SubNavbar>
         <Lorel Ipsum content>
  )

**Review by nikhilrobosoft — APPROVED** [review]

File: components/navbar/navbar.module.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
+.container {
+  height: 80px;
+  background-color: #432c7a;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 10px;
+  color: white;
+
+  ul {
```
Comment (nikhilrobosoft) [inline components/navbar/navbar.module.scss]: Instead of ul add a classname

File: components/navbar/navbar.module.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 50%;
+  }
+
+  li {
```
Comment (nikhilrobosoft) [inline components/navbar/navbar.module.scss]: same instead of li add a classname

File: components/subnavbar/subnavbar.module.scss [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  background-color: #8ec3b0;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 0 10px;
+  margin: 0;
+  color: white;
+
+  li {
+    list-style: none;
```
Comment (nikhilrobosoft) [inline components/subnavbar/subnavbar.module.scss:12]: here too

File: components/navbar/navbar.module.scss (hunk 3 above)
Comment (Anusha-Acharya-99) [reply components/navbar/navbar.module.scss]: added classname to ul

File: components/navbar/navbar.module.scss (hunk 4 above)
Comment (Anusha-Acharya-99) [reply components/navbar/navbar.module.scss]: added classname to li

File: components/subnavbar/subnavbar.module.scss (hunk 5 above)
Comment (Anusha-Acharya-99) [reply components/subnavbar/subnavbar.module.scss:12]: added classname

**Review by nikhilrobosoft — APPROVED** [review]

**Pull request merged**